src/pages/patientView/clinicalInformation/style/local.module.scss.d.ts

```diff
@@ -0,0 +1,3 @@
+
```

Comment:
empty file?

Reply:
yep, we have an empty style file :)

Reply:
Sorry, the empty one was the other file. Good catch!

Here the source scss doesn't really have any named references, it shouldn't be defined as a module but probably a regular scss.

Reply:
seems like we don't even import `local.module.scss` anywhere, probably safe to remove it completely.